velox/type/Type.h

```diff
@@ -2390,6 +2390,16 @@ struct IsRowType<Row<Ts...>> {
 
 } // namespace facebook::velox
 
+namespace std {
+template <>
+struct hash<::facebook::velox::UnknownValue> {
```

Comment:
The hash for UNKNOWN type is required by VectorMakerStats. If lacked, compiling will fail.
See https://github.com/facebookincubator/velox/blob/main/velox/vector/tests/utils/VectorMakerStats.h#L86.

Reply:
VectorMakerStats is deprecated. It would be great to not use it (and eventually delete it).

Reply:
I just removed this and introduced some code in `VectorMakerStats.h` to make the compiling pass. Maybe, better to delete deprecated code in a separate PR.